src/test/kotlin/HelloWorldtests/HelloWorldtest.kt

```diff
@@ -0,0 +1,14 @@
+package HelloWorldtests
+
+import org.junit.Test
+
+class HelloWorldtest{
```

Comment:
The class (and the file) has to be called `HelloWorldTest` (note the upper case `T`) for Maven to pick it up. It worked when you see _Hello from the otherside!!_ on the command line.